google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITBaseTest.java

```diff
@@ -48,6 +65,45 @@ public void before() {
 
     firestoreOptions = optionsBuilder.build();
     firestore = firestoreOptions.getService();
+    primeBackend();
+  }
+
+  public void primeBackend() throws Exception {
+    if (!backendPrimed) {
```

Comment:
I think you should refactor this to be a guard thereby flattening the indentation. It also makes reading easier because you don't have to search for an `else` code block.

```
if (backendPrimed) return;
```

This is one of those contentious situations where I think a one liner without curly braces is acceptable, but I doubt that follows the style guide.